src/modules/tests/test-data/manoeuvres/manoeuvres.reducer.ts

```diff
@@ -1,13 +1,13 @@
-import { Manoeuvres } from '@dvsa/mes-test-schema/categories/B';
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
```

Comment:
Should we call this file `src/modules/tests/test-data/manoeuvres/manoeuvres.cat-b.reducer.ts`?

Reply:
Happy to leave it if we all know no category in file name means its category B